linters.go

```diff
@@ -340,6 +340,13 @@ var defaultLinters = map[string]LinterConfig{
 		InstallFrom:       "github.com/alexkohler/nakedret",
 		PartitionStrategy: partitionPathsAsDirectories,
 	},
+	"predeclared": {
+		Command:           `predeclared`,
+		Pattern:           `PATH:LINE:COL:MESSAGE`,
+		InstallFrom:       "github.com/nishanths/predeclared",
+		PartitionStrategy: partitionPathsAsDirectories,
+		IsFast:            true,
```

Comment:
Using `true` based on the fact that the speed is comparable to ineffassign, which is marked as fast:

```sh
$ time ineffassign /usr/local/go/src/net/
/usr/local/go/src/net/dial_test.go:338:6: ineffectual assignment to err
/usr/local/go/src/net/fd_plan9.go:73:4: ineffectual assignment to err
/usr/local/go/src/net/http/fs.go:604:5: ineffectual assignment to name
/usr/local/go/src/net/http/request_test.go:147:13: ineffectual assignment to err
/usr/local/go/src/net/http/serve_test.go:4452:13: ineffectual assignment to err
/usr/local/go/src/net/http/transport_test.go:716:8: ineffectual assignment to err
/usr/local/go/src/net/http/transport_test.go:2452:3: ineffectual assignment to slurp
/usr/local/go/src/net/interface_plan9.go:155:3: ineffectual assignment to line
/usr/local/go/src/net/interface_plan9.go:155:9: ineffectual assignment to ok
/usr/local/go/src/net/udpsock_plan9.go:80:16: ineffectual assignment to i

real	0m0.401s
user	0m0.378s
sys	0m0.027s
```

```sh
$ time predeclared /usr/local/go/src/net/
/usr/local/go/src/net/file_plan9.go:34:2: variable close has same name as predeclared identifier
/usr/local/go/src/net/http/clientserver_test.go:1025:12: variable close has same name as predeclared identifier
/usr/local/go/src/net/http/server.go:1927:30: param error has same name as predeclared identifier
/usr/local/go/src/net/lookup.go:76:56: named return error has same name as predeclared identifier
/usr/local/go/src/net/mac.go:83:1: label error has same name as predeclared identifier
/usr/local/go/src/net/rpc/server.go:207:2: variable rune has same name as predeclared identifier

real	0m0.292s
user	0m0.229s
sys	0m0.033s
```